api/app/controllers/mno_enterprise/jpi/v1/admin/sub_tenants_controller.rb

```diff
@@ -0,0 +1,70 @@
+module MnoEnterprise
+  class Jpi::V1::Admin::SubTenantsController < Jpi::V1::Admin::BaseResourceController
+
+    # GET /mnoe/jpi/v1/admin/sub_tenants
+    def index
+      # Index mode
+      @sub_tenants = MnoEnterprise::SubTenant
+      @sub_tenants = @sub_tenants.limit(params[:limit]) if params[:limit]
+      @sub_tenants = @sub_tenants.skip(params[:offset]) if params[:offset]
+      @sub_tenants = @sub_tenants.order_by(params[:order_by]) if params[:order_by]
+      @sub_tenants = @sub_tenants.where(params[:where]) if params[:where]
+      @sub_tenants = @sub_tenants.all
```

Comment:
Should we extract all this logic to a separate method on the BaseResource? Or a helper method in the Admin::BaseResourceController?